src/main/resources/templates/application/partials/summary-fragments.html

```diff
@@ -45,3 +45,43 @@
   </label>
   <input type="hidden" th:name="${'_' + id}" value="on">
 </div>
+
+<tr th:fragment="meritsAssessmentFragment" class="govuk-table__row">
+  <td class="govuk-table__cell" th:if="${summary.meritsAssessment.enabled}">
+    <a href="/opa?assessment=merits&invoked-from=CCMS_CA01" class="govuk-link">Merits assessment</a>
+  </td>
+  <td class="govuk-table__cell" th:unless="${summary.meritsAssessment.enabled}">
+    Merits assessment
+  </td>
+  <td class="govuk-table__cell" th:insert="~{application/partials/status-tag :: statusTag(${summary.meritsAssessment.status})}"></td>
+  <td class="govuk-table__cell" th:text="${#dates.format(summary.meritsAssessment.lastSaved, 'dd/MM/yyyy')}"></td>
+  <td class="govuk-table__cell" th:text="${summary.meritsAssessment.lastSavedBy}"></td>
+  <th:block th:if="${summary.meritsAssessment.status} == 'In Progress'">
```

Comment:
Use the enum here?

Reply:
values are different to the enum here, as enum values is the key used for the lookup rather than the value.